integreat_cms/static/src/js/media-management/component/breadcrumbs.tsx

```diff
@@ -1,23 +1,58 @@
 /*
  * This component renders a breadcrumbs list for the current directory and all its parent directories
  */
-import { ChevronRight } from "preact-feather";
+import { ChevronRight, FolderOpen } from "lucide-preact";
 import { Link } from "preact-router";
+import { useState } from "preact/hooks";
 
 import { Directory } from "../index";
 
 interface Props {
   breadCrumbs: Directory[];
   searchQuery: string;
   mediaTranslations: any;
+  allowDrop: boolean;
+  dropItem: (targetDirectory: number) => unknown;
 }
 
-export default function Breadcrumbs({ breadCrumbs, searchQuery, mediaTranslations }: Props) {
+export default function Breadcrumbs({
+  breadCrumbs,
+  searchQuery,
+  mediaTranslations,
+  allowDrop,
+  dropItem,
+}: Props) {
+  const [currentDragTarget, setCurrentDragTarget] = useState<null | number>(null);
   return (
-    <nav className="p-2">
+    <nav className={allowDrop ? "p-5" : "p-2"}>
       <ul class="flex flex-wrap">
-        <li>
-          <Link href={"/"} className={"block hover:bg-water-600 px-3 py-2 rounded"} media-library-link>
+        <li
+          className={currentDragTarget === 0 ? "text-blue-600 text-2xl" : ""}
```

Comment:
```suggestion
          className={currentDragTarget === 0 ? "text-blue-600" : ""}
```
I would leave the text size consistent